bin/libs/fs.sh

```diff
@@ -88,6 +88,17 @@ create_tmp_file() {
   done
 }
 
+###############################
+# Copies dataset to Unix file
+#
+#
+# @param string   Dataset name
+# @param string   Unix file name
+copy_mvs_to_uss() {
+  cp "//'$1'" "$2"
```

Comment:
So this will not work if a PDS is open, even in non-exclusive mode.  cp is just stupid. Zowe V1 has a copy item to PDS that uses a CLIST to do the copying. I copied this and made it more generic for UMS purposes.  I note it does have some issues, in particular it requires the user to have a prefix. But maybe there is some way to rewrite it with some temporary data set that is specified in zowe.yaml.

I can't find it in git hub directly, because I'm sure it's in some older branch. This is where I find it in my V1 installation:
scripts/utils/zowe-install-xmem.sh which calls zowe-copy-to-JES.sh which calls mcopyshr.clist

I can provide the generic script that we created for UMS from this base. It would be great if something like that used everywhere instead of cp. (Especially if the bugs were fixed.)